packages/dao/scripts/deploy.js

```diff
@@ -27,6 +40,21 @@ module.exports = async callback => {
                 {name: 'payroll', contractName: 'Payroll'},
                 {name: 'finance', contractName: 'Finance'},
                 {name: 'token-manager', contractName: 'TokenManager'}]);
+        const tx = await template.newInstance('api3template', (api3Pool.address), [SUPPORT_1, ACCEPTANCE_1, VOTING_DURATION_1], [SUPPORT_2, ACCEPTANCE_2, VOTING_DURATION_2])
```

Comment:
pls change 'api3template' to `TEMPLATE_NAME`

Reply:
Done